backend/subroker/views.py

```diff
@@ -1,3 +1,392 @@
+def content_list(request, content_id):
+    return HttpResponse("content search")
+
+def content_info(request, content_id):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            try:
+                content = Content.objects.get(id=content_id)
+            except (Content.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            user_arr = []
+            users = content.user_id.all()
+            for user in users:
+                user_arr.append(user.id)
+            return JsonResponse({
+                "id": content.id, 
+                "the_movie_id": content.the_movie_id,
+                "name": content.name,
+                "favorite_count": content.favorite_cnt,
+                "user_id": user_arr
+                }, status=200)
+        else:
+            return HttpResponse(status=401)
+
+def recommendation(request, user_id):
+    return HttpResponse("recommendation")
+
+def user_favorite_list(request):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            fav_return = [content for content in Content.objects.filter(user_id__in=[request.user]).values()]
+            if not fav_return:
+                return HttpResponse(status=404)
+            return JsonResponse(fav_return, safe=False, status=200)
+        else:
+            return HttpResponse(status=401)
+
+def content_favorite(request, content_id):
+    if request.method == 'PUT':
+        if request.user.is_authenticated:
+            new_fav = request.user
+            try:
+                content = Content.objects.get(id=content_id)
+            except(Content.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            content.user_id.add(new_fav)
+            content.save()
+            fav_arr = []
+            fav_users = content.user_id.all()
+            for fav_user in fav_users:
+                fav_arr.append(fav_user.id)
+            response_dict = {
+                "id": content.id, 
+                "members": fav_arr
+            }
+            return JsonResponse(response_dict, status=200)
+        else:
+            return HttpResponse(status=401)
+
+    elif request.method == 'DELETE':
+        if request.user.is_authenticated:
+            try:
+                content = Content.objects.get(id=content_id)
+            except(Content.DoesNotExist) as e:
+                return HttpResponse(status=404)
+            content.user_id.remove(request.user)
+            return HttpResponse(status=200)
+        else:
+            return HttpResponse(status=401)
+    else:
+        return HttpResponseNotAllowed(['PUT', 'DELETE'])
+
+def trending(request):
+    return HttpResponse("trending")
+
+#Review
+def review_content(request, content_id):
+    if request.method == 'GET':
+        if request.user.is_authenticated:
+            review_return = [review for review in Review.objects.filter(content__id__startswith=str(content_id)).values()]
```

Comment:
@muhwagwa How about get content first and get reviews using related_name(suppose it is "reviews")
```suggestion
            content = Content.object.get(id=content_id)
            reviews = [review for review in content.reviews.all()]
```